paragon/_variables.scss

```diff
@@ -0,0 +1,2 @@
+@import "../common/variables";
+@import "../lms/static/sass/common-variables";
```

Comment:
Should we import `paragon/_fonts.scss` somewhere here? I know it's empty, but if it exists, then we should import it.

Reply:
Currently this file and _variables are both imported in the MFE's index.scss file separately, and is loaded by all MFEs in this manner.